packages/web/src/components/gcds-card/gcds-card.tsx

```diff
@@ -63,6 +67,30 @@ export class GcdsCard {
    */
   @State() lang: string;
 
+  /**
+   * State to track validation on badge
+   */
+  @State() badgeError: boolean = false;
```

Comment:
I can foresee more errors we would need to introduce in the future, so I think it'll be nice if we can start with a generalized error object, starting here.

Something like:
```
@State() error: Object = null
```
or undefined, basically anything that is falsy

then add as more errors exist like...
```
error = { 'badge': ['the badge attribute has a char limit'] }
```

etc

Reply:
I like that idea! Small addition: can we use `error = { 'gcds-card': ['the badge attribute has a char limit'] }` to make it easier to see which component the error applies to and aligns with the new error message from the stepper?

Reply:
I did something similar to what Daine suggested. Will talk about it with both of you next week